runtime/j9vm/jvm.c

```diff
@@ -336,7 +335,7 @@ static void setLibpath(const char *libpath);
 #ifdef DEBUG_TEST
 static void testBackupAndRestoreLibpath(void);
 #endif /* DEBUG_TEST */
-#endif  /* defined(AIXPPC) || (defined(J9ZOS390) && (JAVA_SPEC_VERSION >= 21)) */
+#endif  /* AIXPPC */
```

Comment:
Since you're here, please correct this comment, and the ones on lines 1298, 1313, 1395, 2189, etc.:
```c
#endif /* defined(AIXPPC) */
```

Reply:
> Since you're here, please correct this comment, and the ones on lines 1298, 1313, 1395, 2189, etc.:

Done.